cpp/ql/src/experimental/Best Practices/UselessTest.ql

```diff
@@ -38,6 +39,8 @@ where
   ) and
   eq.getRightOperand() instanceof Literal and
   ne.getRightOperand() instanceof Literal and
+  eq.getLeftOperand().getFullyConverted().getType() =
+    ne.getLeftOperand().getFullyConverted().getType() and
```

Comment:
I think you'd want to call `getUnspecifiedType` instead of `getType` to resolve `typedef`s and ignore specifiers like `const` and friends.

Reply:
DIdn't know about that, thanks !